openapi/rhose-api.yaml

```diff
@@ -939,6 +943,10 @@ components:
           $ref: '#/components/schemas/Source'
         resolvedAction:
           $ref: '#/components/schemas/Action'
+        isErrorHandler:
```

Comment:
Please ignore me. 

I will stop looking at my phone on holiday where it is clearly impossible to review PRs. Thank-you for continuing with my efforts.